2024-05-02 17:09:24,668 - Debug Adapter (main) - INFO - Working directory: C:\Users\leroy\AppData\Local\Programs\Microsoft VS Code
2024-05-02 17:09:24,668 - Debug Adapter (main) - INFO - Arguments: 
{   'app_flash_off': 65536,
    'board_type': None,
    'cmdfile': 'd:\\repos3\\esp-idf-work\\esp32-thingformation\\blink3\\blink\\esp-idf-vscode-generated.gdb',
    'core_file': (),
    'debug': 3,
    'developer_mode': 'none',
    'device_name': 'esp32c6',
    'elfpath': (   'd:\\repos3\\esp-idf-work\\esp32-thingformation\\blink3\\blink\\build\\blink.elf',),
    'log_file': 'd:\\repos3\\esp-idf-work\\esp32-thingformation\\blink3\\blink\\debug.log',
    'log_mult_files': None,
    'log_no_debug_console': None,
    'oocd': 'openocd',
    'oocd_args': None,
    'oocd_ip': 'localhost',
    'oocd_mode': 'connect_to_instance',
    'oocd_scripts': None,
    'port': 43474,
    'postmortem': False,
    'tmo_scale_factor': 5,
    'toolchain_prefix': 'riscv32-esp-elf-'}
2024-05-02 17:09:24,669 - Debug Adapter (main) - CRITICAL - Debug adapter -> Extension: DEBUG_ADAPTER_STARTED
2024-05-02 17:09:24,669 - Debug Adapter (main) - INFO - Starting. Cmd: c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter_main.py -d 3 -e d:\repos3\esp-idf-work\esp32-thingformation\blink3\blink\build\blink.elf -l d:\repos3\esp-idf-work\esp32-thingformation\blink3\blink\debug.log -p 43474 -dn esp32c6 -a 0x10000 -t riscv32-esp-elf- -x d:\repos3\esp-idf-work\esp32-thingformation\blink3\blink\esp-idf-vscode-generated.gdb -tsf 5

2024-05-02 17:09:24,674 - Debug Adapter (main) - INFO - Listening on port 43474
2024-05-02 17:09:24,674 - Debug Adapter (main) - CRITICAL - Debug adapter -> Extension: DEBUG_ADAPTER_READY2CONNECT
2024-05-02 17:09:24,937 - Gdb - INFO - NOTIFY: thread-group-added {'id': 'i1'}
2024-05-02 17:09:24,969 - Gdb - INFO - CONS: ("Warning: 'set logging on', an alias for the command 'set logging enabled', "
 'is deprecated.\\n')
2024-05-02 17:09:24,969 - Gdb - INFO - CONS: "Use 'set logging enabled on'.\\n\\n"
2024-05-02 17:10:20,517 - Debug Adapter (main) - ERROR - Failed to connect to "remote 127.0.0.1:3333"!
Traceback (most recent call last):
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\command_processor.py", line 52, in __call__
    on_request(protocol_message)
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\command_processor.py", line 75, in on_initialize_request
    raise e
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\command_processor.py", line 70, in on_initialize_request
    self.da.adapter_init()
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\debug_adapter.py", line 329, in adapter_init
    raise e
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\debug_adapter.py", line 325, in adapter_init
    self.start_gdb()
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\debug_adapter.py", line 678, in start_gdb
    raise e
  File "c:\Users\leroy\.vscode\extensions\espressif.esp-idf-extension-1.7.1\esp_debug_adapter\debug_adapter\debug_adapter.py", line 675, in start_gdb
    self._gdb.connect()
  File "D:\Espressif\python_env\idf5.2_py3.11_env\Lib\site-packages\esp_debug_backend\gdb.py", line 599, in connect
    self.target_select(remote_mode, self._remote_target, tmo=tmo)
  File "D:\Espressif\python_env\idf5.2_py3.11_env\Lib\site-packages\esp_debug_backend\gdb.py", line 265, in target_select
    raise DebuggerError('Failed to connect to "%s %s"!' % (tgt_type, tgt_params))
esp_debug_backend.defs.DebuggerError: Failed to connect to "remote 127.0.0.1:3333"!
2024-05-02 17:10:20,540 - Debug Adapter (main) - CRITICAL - Debug adapter -> Extension: DEBUG_ADAPTER_STOPPED
